src/OpenTracing/Span.php

```diff
@@ -32,28 +35,33 @@ public function overwriteOperationName($newOperationName);
      * As an implementor consider supporting a single Tag object or a $tag, $tagValue.
      *
      * @param array $tags
+     * @return static
+     * @throws SpanAlreadyFinished if the span is already finished
      */
-    public function addTags(array $tags);
+    public function withTags(array $tags);
```

Comment:
@felixfbecker I believe this should be `addTags` as it was since `setTags` might not be clear. If an span does contain tags and I call the `setTags` method, will it override the existing ones or add the new ones?